groceries_app/static/css/grocery-planner/gp1-meal-choice.css

```diff
@@ -0,0 +1,53 @@
+a {
+    text-decoration: none;
+}
+
+a:hover {
+    text-decoration: none;
+}
+
+.jumbotron {
+    margin-bottom: 96px;
+}
+
+.card {
+    border: 3px solid;
+}
+
+.card-header {
+    border-bottom-width: 0;
+}
+
+.meal-name-column {
+    text-align: left;
+}
+
+#page-title {
+    max-width: 20rem;
+    border: 4px ridge black;
+}
+
+#checkout-footer {
+    position:fixed;
+    bottom:0;
+    left:0;
+    width: 100%;
+    background-color: white;
+}
+
+#footer-card {
+    max-width: 100%;
+    margin: 0px 1px 4px 1px;
+    border-radius: 5px 5px 5px 5px/3px 3px 25px 25px;
```

Comment:
What do these 2 sets of values mean?

Reply:
border-radius works with how the corners of a box shaped border look. The first setting how rounded the corners will be. The second set modifies how the sides transition into the curve, in this case making the curves almost look hand-drawn. 

https://www.w3schools.com/cssref/tryit.asp?filename=trycss3_border-radius5
This page can be fun to play with to see how it can work.